### PR TITLE
Updated rollbar to v2.2.8

### PR DESCRIPTION
**Pull request by boyanyordanov**
I also added a test to verifiy that `checkIgnore` works correctly. 
It is probably not necessary for a simple update but it does verify that the callback works correctly after the version bump and fails before that.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "broccoli-funnel": "2.0.1",
     "ember-cli-babel": "6.8.2",
-    "rollbar": "~2.2.7"
+    "rollbar": "~2.2.8"
```
Comment (Exelord) [inline package.json]: We don't have to bump it as we have `~` which will automatically resolve `rollbar` to the latest `2.2.*` version

File: package.json [hunk 2]
```diff
@@ -50,7 +50,6 @@
     "ember-load-initializers": "1.0.0",
     "ember-resolver": "4.5.0",
     "ember-scroll-to": "0.6.4",
-    "ember-source": "~2.15.0",
```
Comment (Exelord) [inline package.json]: Why ember was removed?

File: tests/unit/services/rollbar-test.js [hunk 3]
```diff
@@ -95,4 +95,27 @@ test('registerLogger: does not register logger if disabled', function(assert) {
   };
   service.registerLogger();
   Ember.onerror();
+});
+
+test('config: rollbar passes arguments to checkIgnore callback', function (assert) {
```
Comment (Exelord) [inline tests/unit/services/rollbar-test.js]: This is not our responsibility to check if it works. But yeah... we should definitely have a test which will check if a proper config has been passed to the rollbar. That's it.

So, we should check if our options from config has been passed to `rollbarService.get('notifier.options')`

File: package.json (hunk 2 above)
Comment (boyanyordanov) [reply package.json]: Wow, I don't think that's me and did not see it. Sorry.

File: package.json (hunk 1 above)
Comment (boyanyordanov) [reply package.json]: It still installed 2.2.7 this morning. It installs 2.2.8 now, so thanks!

**Comment by boyanyordanov** [comment]
@Exelord I think at this point we can close this PR. 
I can make a new one adding the test for the options object if you wish.

**Comment by Exelord** [comment]
That would be great! Thanks :)

**Comment by sdhull** [comment]
I've stumbled on this because I've found that `checkIgnore` isn't firing for me.

I added a `debugger` to the top of my `checkIgnore` function and it is never triggered. I'm a bit fuzzy on how exactly the config values end up being passed to Rollbar.js so I'm not really sure where to start looking to try to fix it

**Comment by Exelord** [comment]
We are passing them directly but from what I remember we are missing a test for that :/
Theoretically, if any property from options works and the `checkIgnore` is not fired, you should check out Rollbar issues, as this is only a wrapper for it, not a full ember implementation.

**Comment by sdhull** [comment]
In my compiled app, where would I find the definition of my `checkIgnore` function? Would it be injected into `index.html`, compiled to `vendor.js`, in `my-app.js` or... somewhere else?

**Comment by Exelord** [comment]
It depends from your configuration. Normally it should be in your head section in config part

**Comment by sdhull** [comment]
OK that's kinda what I was thinking. It appears that the config hash is jsonified & then uri encoded, it's pretty clear that `function` is not part of the json standard and cannot survive being jsonified.
Right?

**Comment by Exelord** [comment]
You're right! Let's create a new issue for that.

**Pull request closed without merging**